### PR TITLE
add description for chopsol/contao-matomo-tracker

### PR DESCRIPTION
**Pull request by chopsol**

### CODE REVIEW

File: meta/chopsol/contao-matomo-tracker/de.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+de:
+  title: Besuchertracking mit Matomo
+  description: >
+    Tracking aller Besuchern der Webseite direkt aus Contao über die API von Matomo ohne Javascript.
```
Comment (xchs) [inline meta/chopsol/contao-matomo-tracker/de.yml:4]:
```suggestion
    Tracking aller Besucher der Website direkt aus Contao über die API von Matomo ohne JavaScript.
```

File: meta/chopsol/contao-matomo-tracker/en.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+en:
+  title: Visitortracking with Matomo
+  description: >
+    Tracking of all visitors to the website directly from Contao via the Matomo API without Javascript.
```
Comment (xchs) [inline meta/chopsol/contao-matomo-tracker/en.yml:4]:
```suggestion
    Tracking of all website visitors directly from Contao via the Matomo API without JavaScript.
```

File: meta/chopsol/contao-matomo-tracker/en.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+en:
+  title: Visitortracking with Matomo
```
Comment (xchs) [inline meta/chopsol/contao-matomo-tracker/en.yml:2]:
```suggestion
  title: Visitor tracking with Matomo
```

**Comment by xchs** [comment]
As the lint process is failing you would have to add

* `Besuchertracking` to the [German whitelist](https://github.com/contao/package-metadata/blob/master/linter/whitelists/de.txt) and
* `Matomo` to the [Default whitelist](https://github.com/contao/package-metadata/blob/master/linter/whitelists/default.txt)

**Comment by xchs** [comment]
@chopsol Before we can merge this, you would have to make the mentioned changes.

**Comment by Toflar** [comment]
I've removed the `master` branch and renamed it to `main`. Please target it to `main` if you still want to get this merged.

**Pull request closed without merging**